frontend/src/components/RiderCard/RiderCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { RiderTableValue } from '../../types/index';
+import styles from './card.module.css';
+
+
+type RiderCardProps = {
+  values: Array<RiderTableValue>;
+};
+
+const RiderCard = (props: RiderCardProps) => {
+  const { values } = props;
+
+  const resultList = values.map((val, index) => {
+    const { data, tag, buttonHandler } = val;
+    if (index === 0) { /* first cell */
+      return (<td key={index} className={`${styles.passInfo} ${styles.cell} 
```

Comment:
there should be a package called 'classnames' that lets you chain classnames cleanly:
```
import cn from 'classnames';
<td className={cn(styles.passInfo, styles.cell, styles.firstCell)}>
```
also since these <td> elements are pretty similar to teach other, you could create a component for it and add boolean props to determine if its first, last, or neither
and if you have a really long jsx element being returned, i like to format it like this to make it more readable
```
return (
  <td
    key={index}
    className={cn(styles.passInfo, styles.cell, styles.firstCell)}
  >
    {data}
  </td>
)
```